spec/models/todo_spec.rb

```diff
@@ -1,5 +1,22 @@
 require 'rails_helper'
 
-RSpec.describe Todo, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
-end
+describe 'todo-method' do
+  before do
+    @user = create(:user)
```

Comment:
no need to create a user here. the todo factory will create one behind the scenes, for the associated user